### ISSUE TITLE
Some actions not fitting their correct syntax fail silently and timeout

### ISSUE THREAD

**Issue by antoniojl** [pa11y/pa11y#396]
## Expected behaviour

`'wait for path not to be /account/signin/'` <- should error out

## Actual behaviour

`'wait for path not to be /account/signin/'` <- fails silently
`'wait for path to not be /account/signin/'` <- works
`'wait for path woooooohooooooo to be /account/signin/'` <- fails silently

## Steps to reproduce

Create a pa11y test config with the above

**Comment by rowanmanning** [comment]
Thanks @antoniojl, this should definitely fail and it may be that something was missed in the switch to Pa11y 5. I'll investigate

**Comment by rowanmanning** [comment]
OK what's happening is the greedy match in the regexp is causing issues. I'm going to have a think about how to fix this without a breaking change to Pa11y. It's possible that we just need better debug logs for possibly-malformed actions.

In your examples above, the following Regexp is used:
```js
/^wait for (fragment|hash|host|path|url)( to (not )?be)? (.+)$/i
```
And the following is being parsed out:

**Input:** `wait for path to not be /account/signin/`
**Read as:** `{subject: 'path', negated: true, value: '/account/signin'}`

**Input:** `wait for path not to be /account/signin/`
**Read as:** `{subject: 'path', negated: false, value: 'not to be /account/signin/'}`

**Input:** `wait for path woooooohooooooo to be /account/signin/`
**Read as:** `{subject: 'path', negated: false, value: 'woooooohooooooo to be /account/signin/'}`

**Comment by josebolos** [comment]
This should be fixed in pa11y 5.2.0.
